src/components/Footer/index.jsx

```diff
@@ -0,0 +1,153 @@
+// import Button
+import Image from "next/image";
+import Link from "next/link";
+import { useTranslation } from "next-i18next";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import React from "react";
+import {
+    FaFacebookF,
+    FaInstagram,
+    FaLinkedin,
+    FaTwitter,
+    FaYoutube,
+} from "react-icons/fa";
+
+const Footer = () => {
+    const current_year = new Date().getFullYear();
+    const { t } = useTranslation("footer");
+    return (
+        <div>
+            <div className='container mx-auto text-white bg-[#43AFD6] py-6'>
+                <div className='grid grid-cols-1 sm:grid-cols-1 md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-3 gap-4'>
+                    <div className='space-y-3'>
+                        <div>
+                            <Image
+                                src='/footer_icons/logo.svg'
+                                alt='logo-image'
+                                width='40'
+                                height='70'
+                            />
+                        </div>
+                        <div>
+                            <h1 className='text-3xl'>
+                                <b>Food on coupon</b>
+                            </h1>
```

Comment:
Great job @mohamed70601 on creating the Footer, However:

can you replace that line with LogoFood.svg it's already in the project file to match the font logo in figma.

Excellent work, and keep it up.